src/my_std.ml

```diff
@@ -275,13 +275,80 @@ let sys_file_exists x =
       try Array.iter (fun x -> if x = basename then raise Exit) a; false
       with Exit -> true
 
+(* https://github.com/ocaml/opam/blame/master/src/core/opamStd.ml *)
```

Comment:
(Luckily opam uses the same license so reusing code as-is is just fine.)

@hhugo could you maybe tweak your URL to contain a hash of a recent commit, instead of `master`? This would be helpful if someone in the future wants to now whether our local copy should be updated.

(I would have considered moving the copied-almost-exactly-as-is code to a separate file, to make such future tracking easier, but oh well.)

Reply:
I've updated the comment